src/main/java/seedu/address/ui/ContactDetailsDisplay.java

```diff
@@ -43,15 +45,22 @@ public ContactDetailsDisplay() {
         registerAsAnEventHandler(this);
     }
 
+    private Image getImage(String imagePath) {
+        return new Image(MainApp.class.getResourceAsStream(imagePath));
+    }
+
     /**
      *Shows the contact details of the person
      */
     private void showPersonDetails(Person person) {
 
         name.setWrapText(true);
         name.setText(person.getName().fullName);
-        imageView.setImage(person.getProfilePicture().getImage());
-
+        if (person.getProfilePicture().filePath != null) {
+            imageView.setImage(person.getProfilePicture().getImage());
+        } else {
+            imageView.setImage(getImage(DEFAULT_IMAGE));
```

Comment:
This shows the default image even if no person is selected. Any way to fix that?

Reply:
The bug is due to a change i make in fxml file. I have fixed it.